### ISSUE TITLE
@primer/octicons-react: defaultProps is deprecated

### ISSUE THREAD

**Issue by nanaian** [primer/octicons#884]
`defaultProps` was recently deprecated in React: see https://github.com/facebook/react/pull/16210.

Using any Octicon produces something like the following warning:

> Warning: UploadIcon: Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.

This is caused by the following code, which needs to be swapped out:

https://github.com/primer/octicons/blob/e05ddd3317c460c1e298cb232bdaa4c2a2d863de/lib/octicons_react/src/createIconComponent.js#L45

**Comment by Angelfire** [comment]
That's right, I'm getting the same error, I think the solution is easy, just use default parameters in the `Icon` function, something like:
```js
Icon({'aria-label': ariaLabel, tabIndex, className, fill = 'currentColor', size = 16, verticalAlign = 'text-bottom'})
```

Or something like that...
